### PR TITLE
{bio}[gompi/2019b] MIRA v4.0.2

### PR DESCRIPTION
**Pull request by tstrempel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by tstrempel** [comment]
new dep for https://github.com/easybuilders/easybuild-easyconfigs/pull/10811

File: easybuild/easyconfigs/m/MIRA/MIRA-4.0.2_fix-ads-include.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- src/mira/ads.C.orig	2020-06-22 14:05:17.506971632 +0200
```
Comment (smoors) [inline easybuild/easyconfigs/m/MIRA/MIRA-4.0.2_fix-ads-include.patch:1]: @tstrempel please add short description and author to the top of the patch?

**Comment by smoors** [comment]
@tstrempel now that you updated the patch, its checksum should be updated too.

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 3 out of 5 (3 easyconfigs in this PR)
node378.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/9cd7b077657a93fb9396c13ed6ba4379 for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 5 out of 5 (3 easyconfigs in this PR)
node365.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/0bc778e13a4ec4d6a112444e3cb04731 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @tstrempel!

**Pull request merged**